Usage.md

```diff
@@ -286,7 +286,7 @@ mvn verify -Dtarget=stage01 -Dprofile=smoke
 
 ## Authenticated Calls
 
-Often REST API calls will need to be Authenticated.  However, authentication schemes vary widely by company.  We have tried to make allowances for authentication by add arguments about whether to add a token to a service call, or not.  In the Ride core, there is a check which utilizes a rest-assured Filter class.  This checks to see if there is already an Authorization header added to the call, and if it is not present, it attempts to add one.  However, again, and with good reason, these company-specific authentication details are kept secret and you will need to do a bit of work here to determine how to add authorization to your call.
+Often REST API calls will need to be Authenticated.  However, authentication schemes vary widely by company.  We have tried to make allowances for authentication by adding arguments about whether to add a token to a service call, or not.  In the Ride core, there is a check which utilizes a rest-assured Filter class.  This checks to see if there is already an Authorization header added to the call, and if it is not present, it attempts to add one.  However, again, and with good reason, these company-specific authentication details are kept secret and you will need to do a bit of work here to determine how to add authorization to your call.
```

Comment:
This is another one of those examples where it's hard to tell what changed.

Reply:
Sorry about that, I'm not sure why it didn't highlight the change. In this case it was just changing "by add arguments" to "by adding arguments" in the second sentence.